packages/jaeger-ui/src/components/SearchTracePage/SearchResults/DiffSelection.test.js

```diff
@@ -46,23 +47,31 @@ describe('DiffSelection', () => {
   ];
 
   it('renders a trace as expected', () => {
-    const wrapper = shallow(
-      <DiffSelection traces={traces.slice(0, 1)} toggleComparison={toggleComparison} />
+    const { container } = render(
+      <MemoryRouter>
```

Comment:
is MemoryRouter the recommended & forward looking way when we need to upgrade 'react-router-dom'?